### ISSUE TITLE
Bring "Open in Windows Terminal" for Right Click on Drive Letter

### ISSUE THREAD

**Issue by MikeMcCollister** [microsoft/terminal#6654]
# Description of the new feature/enhancement

Bring "Open in Windows Terminal" for right click on drive letter. In Windows Terminal 1.1.1671.0 I am able to do this on a folder but it does not work on a drive letter.

**Comment by zadjii-msft** [comment]
I'm not _sure_ if this is caused by the same thing as #6414 or not.

**Comment by MikeMcCollister** [comment]
I'm glad your tracking that issue as well (#6414). I was about report that.

**Comment by DHowett** [comment]
No no, this is the exact same thing. /dup #6414

**Comment by DHowett** [comment]
That should teach me to read. It probably _isn't_ the same.

**Comment by MikeMcCollister** [comment]
I'm now using version 1.6.10571.0 of Windows Terminal. There still is no right click on the drive letter to "Open in Windows Terminal". I can, however, right click on a white area in any folder and "Open in Windows Terminal" is available.

After reading this thread, I'm confused on whether this is a duplicate of another issue. When I searched around, I came right back to this one.

**Comment by zadjii-msft** [comment]
Heck, I think even Dustin is confused 😆 

![image](https://user-images.githubusercontent.com/18356694/109828376-3079ce80-7c02-11eb-812c-8b07e506b574.png)


I _think_ this is a unique enough issue. I'm not sure how we attach to the context menu for a drive (not just a folder), but _there's gotta be a way_.

**Comment by MikeMcCollister** [comment]
Yep, there has to be a way. With a shift right click there is an "Open PowerShell window here" and on a regular right click there is an "Open with Code" item.

**Comment by zadjii-msft** [comment]
This is being tracked internally with MSFT:32135340 (I think?)

**Comment by DHowett** [comment]
(It cannot be done with packaged COM today. Yeah.)

**Comment by zadjii-msft** [comment]
* MSFT:32135340 has the most notes but the following are also relevant:
* MSFT:35558552
* MSFT:36784232

**Comment by zadjii-msft** [comment]
> Ah. So, I'll need to wait for the SV2 SDK to use this then presumably? I just tried adding this to the Terminal (built on the 22000 SDK) and it gave me the classic APPX0501 error.
> 
> ```xml
>                 <desktop5:ItemType Type="Drive">
>                     <desktop5:Verb Id="OpenTerminalDev" Clsid="52065414-e077-47ec-a3ac-1cc5455e1b54" />
>                 </desktop5:ItemType>
> ```
> 
> Also, will this just gracefully continue to do nothing on Windows 10?
> 
> (I also just tried this with `desktop10:ItemType` but got something about ```The schema for MaxVersionTested specified does not recognize XML fields with namespace "http://schemas.microsoft.com/appx/manifest/desktop/windows10/10".```)

> right, I think it is available in the {SV2} SDK. I think one of them either Drive or DesktopBackground will just work in SV1. I'm not sure about what will happen on Windows 10.

Okay, so if we target the sv2 sdk, this might just work today, but we _really really_ need to confirm on Windows 10 manually.

**Comment by nhchmg** [comment]
> 



> > Ah. So, I'll need to wait for the SV2 SDK to use this then presumably? I just tried adding this to the Terminal (built on the 22000 SDK) and it gave me the classic APPX0501 error.
> > ```
> >                 <desktop5:ItemType Type="Drive">
> >                     <desktop5:Verb Id="OpenTerminalDev" Clsid="52065414-e077-47ec-a3ac-1cc5455e1b54" />
> >                 </desktop5:ItemType>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Also, will this just gracefully continue to do nothing on Windows 10?
> > (I also just tried this with `desktop10:ItemType` but got something about `The schema for MaxVersionTested specified does not recognize XML fields with namespace "http://schemas.microsoft.com/appx/manifest/desktop/windows10/10".`)
> 
> > right, I think it is available in the {SV2} SDK. I think one of them either Drive or DesktopBackground will just work in SV1. I'm not sure about what will happen on Windows 10.
> 
> Okay, so if we target the sv2 sdk, this might just work today, but we _really really_ need to confirm on Windows 10 manually.

  What is sv2 sdk?  does support   Type="Drive" now? please tell me,thank you.

**Comment by AndromedaMelody** [comment]
@nhchmg `sv2 sdk` in appx/msix packaging is [Microsoft.Windows.SDK.BuildTools (Version 10.0.22621.*)](https://www.nuget.org/packages/Microsoft.Windows.SDK.BuildTools/10.0.22621.756/). Until now this is an undocumented property. We have tried to add it in M2Team/NanaZip#313 .

**Comment by xmha97** [comment]
+1
